src/components/DataCollector/index.js

```diff
@@ -57,7 +57,7 @@ const createDataCollector = ({ config, logger }) => {
     const { viewStart = false, documentUnloading = false, xdm, data } = options;
 
     return lifecycle
-      .onBeforeEvent(event, options, viewStart, documentUnloading)
+      .onBeforeEvent({ event, options, viewStart, documentUnloading })
```

Comment:
It looks like the name `viewStart` is inconsistent with `isViewStart` that's being referenced in, for example, `Audiences/index.js`